### ISSUE TITLE
Delete segment

### ISSUE THREAD

**Issue by normanrz** [scalableminds/webknossos#7408]
## Detailed Description
Add a button to the right-click menu to delete a segment. This should overwrite all associated voxels with 0. This would only work for annotation with segment index.

See https://forum.image.sc/t/delete-segment-option-in-volume-annotation-not-available/87838/2

**Comment by philippotto** [comment]
For large segments, I imagine this to be quite inefficient when everything is looped through the front-end. The back-end might be more suitable for the job (the front-end could simply reload the segmentation layer afterwards). @fm3 What do you think?

**Comment by fm3** [comment]
I agree, the backend could also do this job. However, it is currently notoriously inefficient with voxel-wise operations. There are some ideas for that though. Compare #6248
